CRNotifications/Classes/CRNotification.swift

```diff
@@ -164,6 +164,15 @@ public class CRNotificationView: UIView, CRNotification {
         })
     }
     
+    @objc internal func dismissNotificationOnTap() {
```

Comment:
I think its fine to just reuse the current method `dismissNotification` and simply call the delegate, followed by the normal procedure. Otherwise you will have two different implementations of the dismiss functionality and one might not hide correctly 🙂 

Swift will just call the function on the delegate if it's there. There is no need to guard it.
Simply just write `onClickDelegate?.onNotitificationTap()` 🙂

Reply:
as I wrote below, I'd like to handle tapping on the notification differently than just sliding it away. Maybe it should be renamed to dismissNotification and tapNotification? or expandNotification even though this might suggest some more functionality.

Thank you for the Swift tip, you are right!